internal/util/strings.go

```diff
@@ -22,3 +25,12 @@ func ReturnParsedEntityID(entityID string) (*model.ParsedEntityID, error) {
 		Type: s[0],
 	}, nil
 }
+
+func RandomString(length int) string {
+	b := make([]byte, length)
+	for i := range b {
+		b[i] = alphaNumeric[rand.Intn(len(alphaNumeric))]
```

Comment:
I know this is silly but is there a package that does this and filters out any bad words?

Reply:
tracking here: https://chatham.atlassian.net/browse/CHAT-60